tests/test_module.py

```diff
@@ -1477,19 +1501,16 @@ def test_dump_trimmed_series(self):
     def test_ooo(self):
         with self.redis() as r:
             quantity = 50001
-            to_dbl = 1.001
             type_list = ['', 'UNCOMPRESSED']
             for chunk_type in type_list:
-                #r.execute_command('ts.create', 'no_ooo', chunk_type)
-                #r.execute_command('ts.create', 'ooo', chunk_type)
                 r.execute_command('ts.create', 'no_ooo', chunk_type, 'CHUNK_SIZE', 100, 'DUPLICATE_POLICY', 'BLOCK')
                 r.execute_command('ts.create', 'ooo', chunk_type, 'CHUNK_SIZE', 100, 'DUPLICATE_POLICY', 'LAST')
                 for i in range(0, quantity, 5):
-                    r.execute_command('ts.add no_ooo', i, i * to_dbl)
+                    r.execute_command('ts.add no_ooo', i, i)
```

Comment:
@danni-m why don't you want to test double values anymore?

Reply:
@gkorland 
1. its a code-smell, if we want to make it a double/float there's better ways to do that.
2. I don't see it as a crucial piece of the test.